namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-repos/resources/repos.tf

```diff
@@ -191,3 +191,26 @@ resource "kubernetes_secret" "ecr-repo-fb-user-datastore-api" {
     secret_access_key = "${module.ecr-repo-fb-user-datastore-api.secret_access_key}"
   }
 }
+
+##################################################
+
+# User Filestore ECR Repos
+module "ecr-repo-fb-user-filestore-api" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=1.0"
```

Comment:
`?ref=2.1` please, see https://github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials/releases

Reply:
Presumably all the other refs in this file need to be updated from `1.0` to `2.1`?

Reply:
👍